meilisearch-core/src/serde/serializer.rs

```diff
@@ -320,13 +321,17 @@ where
             indexer,
             document_id,
         };
-        if let Some(number_of_words) = value.serialize(indexer)? {
-            documents_fields_counts.put_document_field_count(
-                txn,
-                document_id,
-                attribute,
-                number_of_words as u64,
-            )?;
+        match value.serialize(indexer) {
+            Ok(Some(number_of_words)) => {
+                documents_fields_counts.put_document_field_count(
+                    txn,
+                    document_id,
+                    attribute,
+                    number_of_words as u64,
+                )?;
+            },
+            Ok(None) => (),
+            Err(err) => warn!("Indexing error; {}", err),
```

Comment:
Please revert that to the previous `if let Some`.

Reply:
By doing this you discard every potential errors even those related to write to disk in LMDB.

Reply:
Ok, I got it